integrations/concat/integration/Categorifier/Hierarchy/ConCat.hs

```diff
@@ -54,14 +60,22 @@ hierarchy' moduleName = do
   bottomV <-
     pure <$> do
       fn <- identifier' "bottomC"
-      pure (\onDict cat a b -> mkMethodApps onDict fn [cat, a, b] [] [])
+      pure
+        ( \onDict cat a b ->
+            mkMethodApps
+              onDict
+              fn
+              [Plugins.typeKind a, Plugins.typeKind b, cat, a, b]
+              []
+              []
+        )
```

Comment:
Whoops. Had this just been wrong forever?

Reply:
But the tests didn't fail before, which means for some reason `bottomC` was considered mono-kinded before but poly-kinded now, and I have no idea why..